ttnn/cpp/ttnn/device.cpp

```diff
@@ -9,7 +9,7 @@ namespace ttnn {
 namespace device {
 
 Device &open_device(int device_id, size_t l1_small_size, size_t trace_region_size) {
-    tt::DevicePool::initialize({device_id}, 1, l1_small_size, trace_region_size);
+    tt::DevicePool::initialize({device_id}, 1, l1_small_size, trace_region_size, {}, true);
```

Comment:
what those `{}` and `true` are?

Reply:
`l1_small_remap` and `skip_remote_device flag`.

Reply:
Do you have full control over this code? 
Can we change `skip_remote_device` from bool to enum instead?

Reply:
I added the DevicePool, so I could. Any reason you want an enum over a bool?

Reply:
I could also have it be inferred in the DevicePool init logic. So if device_ids is size 1 and requested device is mmio, I'll just skip by default. This is making more sense to me as I'm typing it out, so I think I'll make that change